fabfile.py

```diff
@@ -92,6 +92,7 @@ def decrypt_secrets(c):
 def install_deps(c: Connection):
     print('-- Installing dependencies')
     with c.cd(c.release_path):
+        # TODO: Edit out pipenv dependency
         c.run('source .venv/bin/activate && pipenv install --deploy', echo=True, env={'PIPENV_VENV_IN_PROJECT': 'true'})
```

Comment:
Here I think you just need to reach the pipenv command with `pip install -r requirements.txt`, and if the dev and prod requirements are split up then that'll do the same thing to only install the prod ones and skip any dev ones.

Reply:
Resolving this convo to have everything in the [other duplicate thread](https://github.com/compserv/hknweb/pull/272/files?file-filters%5B%5D=.html&file-filters%5B%5D=.md&file-filters%5B%5D=.py&file-filters%5B%5D=.txt&file-filters%5B%5D=.yml&file-filters%5B%5D=dotfile&hide-deleted-files=true&file-filters%5B%5D=.lock#r522728333)